webpack.config.js

```diff
@@ -57,6 +57,20 @@ function jsonTransform(emojis) {
   return result;
 }
 
+function jsonTransformCategories(emojis) {
+  const result = {};
+  for (const emojiData of Object.values(emojis)) {
+    const char = convertEmojiToolkitCodePointToChar(emojiData.code_points.fully_qualified);
+    const data = {
+      char,
+      slug: emojiData.shortname.replace(/:/g, ''),
+    };
+
+    result[emojiData.category] = {...result[emojiData.category], [data.slug]: data};
```

Comment:
it would be great to avoid iterating over the same dataset twice. can we re-use the same loop to generated these both?